oauthenticator/tests/test_generic.py

```diff
@@ -166,14 +176,24 @@ async def test_generic(
     c.GenericOAuthenticator = Config(class_config)
     c.GenericOAuthenticator.username_claim = "username"
     authenticator = get_authenticator(config=c)
+    manage_groups = False
+    if "manage_groups" in class_config:
+        try:
+            manage_groups = authenticator.manage_groups
+        except AttributeError:
+            pytest.skip("manage_groups requires jupyterhub 2.2")
+            1 / 0
```

Comment:
Would this be clearer?
```suggestion
    if "manage_groups" in class_config:
        pytest.importorskip("jupyterhub", minversion="2.2", reason="manage_groups requires jupyterhub 2.2")
        manage_groups = authenticator.manage_groups
```